### ISSUE TITLE
Add more bus routes to tracking

### ISSUE THREAD

**Issue by devinmatte** [transitmatters/gobble#86]
YEAR OF THE BUS, let's add a bunch more of the bus routes that don't share a schedule with any other bus routes

- [x] line-7
- [x] line-8
- [x] line-10
- [x] line-11
- [x] line-14
- [x] line-18
- [x] line-26
- [x] line-29
- [x] line-30
- [x] line-31
- [x] line-35
- [x] line-36
- [x] line-37
- [x] line-38
- [x] line-42
- [x] line-43
- [x] line-44
- [x] line-51
- [ ] line-100
- [ ] line-101
- [ ] line-105
- [ ] line-106
- [ ] line-108
- [ ] line-110
- [ ] line-112
- [ ] line-119
- [ ] line-134
- [ ] line-195
- [ ] line-215
- [ ] line-230
- [ ] line-236
- [ ] line-238
- [ ] line-240
- [ ] line-354
- [ ] line-429
- [ ] line-455
- [x] line-69
- [x] line-80
- [x] line-83
- [x] line-87
- [x] line-88
- [x] line-90
- [x] line-93
- [x] line-94
- [x] line-95
- [x] line-96
- [x] line-97
- [x] line-99

**Comment by MoralCode** [comment]
Is there a particular process you followed to add those bus routes? it looks to me like that constants file is just a mapping of bus route numbers to stop numbers. Is this something that has to be manually figured out by physical visit?

**Comment by devinmatte** [comment]
Nope, if you reference other PRs that's all you need. Gobble is the first step to adding it to the dashboard. There is more complex work that requires AWS access before it goes live in the dashboard, but we can't do that until ~1 month after gobble changes merged

**Comment by MoralCode** [comment]
so for example to add like 100 i would go to https://www.mbta.com/schedules/100/line and make a dictionary of all the stop IDs on that line?

(was going to use line 7 as an example but it seems like the stops have changed since your PR - the website shows 18 stops and [the dict](https://github.com/transitmatters/gobble/blob/80341b310b3d289aaaea8b761ef2967e4f825270/src/constants.py#L22) has less than that)

**Comment by MoralCode** [comment]
if that is in fact true, i have found a way to dynamically update this using MBTAs own API (the same api that webpage uses.

`curl 'https://www.mbta.com/schedules/line_api?id=7&direction_id=0&route_pattern=7-2-0' --compressed | jq '{ids: [.route_stop_lists[][] | .station_info.id]}' this will return a JSON object like this:

```json
{
  "ids": [
    "16535",
    "6564",
    "888",
    "889",
    "244",
    "246",
    "208",
    "885",
    "33"
  ]
}
```

**Comment by MoralCode** [comment]
update:

i have found a way to just grab all the patterns for a given bus line and the stop ids for that pattern (as well as the rest of the data but for this purpose i'm ignoring it)

data sample for line 101: [101-patterns.json](https://github.com/user-attachments/files/16022317/101-patterns.json)

**Comment by MoralCode** [comment]
[existing_route_constants.json](https://github.com/user-attachments/files/16042134/existing_route_constants.json)
[new_routes_constants.json](https://github.com/user-attachments/files/16042135/new_routes_constants.json)

Here is the JSON data from all the routes that have been checked off (existing) and not yet checked off (new) in this PR (except line [195](https://www.mbta.com/schedules/195/line) which i noticed has no routes). Both files should be in the same data format as the constants.py file in this PR. 

Once I have a better understanding of whether it would be helpful, I'd be happy to integrate the code that generated these into the main gobble script so it can automatically grab all the stops (or just the stops on the typical route) for all the bus lines on startup to ensure that the data feeding the rest of the TransitMatters dashboards is as complete as possible
